src/routes/console/project-[project]/databases/database-[database]/collection-[collection]/document-[document]/attributeForm.svelte

```diff
@@ -37,7 +37,7 @@
 {#if attributes.length}
     <ul class={`form-list u-gap-${gap}`}>
         {#each attributes as attribute}
-            {@const label = attribute.required ? `${attribute.key}*` : attribute.key}
+            {@const label = attribute.key}
```

Comment:
Not sure we wanna remove the asterik for required values, can you double check with design? 🙂

Reply:
It isn't removed, since we add the required class to the label, which adds a red asterisk instead of the text-color asterisk we had before